Gemfile.lock

```diff
@@ -97,7 +97,7 @@ GEM
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
       ruby_dep (~> 1.2)
-    loofah (2.6.0)
+    loofah (2.7.0)
```

Comment:
### [loofah](https://github.com/flavorjones/loofah)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.6.0 → 2.7.0

[_[change-log](https://github.com/flavorjones/loofah/blob/master/CHANGELOG.md), [source-code](https://github.com/flavorjones/loofah)_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/flavorjones/loofah/compare/v2.6.0...v2.7.0).

These are the individual commits:
- (1ce8698) [update json dev dependency](https://github.com/flavorjones/loofah/commit/1ce86983e546dc8acb34c56e525e78a7cd6019cb)
- (50931f4) [scrub_css should not drop `!important` from shorthand css pr…](https://github.com/flavorjones/loofah/commit/50931f4a4309f34bc15daf4ff2f3d649e7972448)
- (ea6fe90) [prefer Array#<< to creating a new array](https://github.com/flavorjones/loofah/commit/ea6fe90fb6f31e1457b79444511584a872570e51)
- (0aba0b9) [update CHANGELOG](https://github.com/flavorjones/loofah/commit/0aba0b996e0090fc6e4dde3477eb4dd3edba80aa)
- (39e105c) [Merge pull request #192 from b7kich/maintain_shorthand_css_i…](https://github.com/flavorjones/loofah/commit/39e105c1208b07cd99fce749ebc45b3a5b228220)
- (c0f4bb8) [add page-break to safelist](https://github.com/flavorjones/loofah/commit/c0f4bb86c23dd29e8c6c45dbbd0f7313a5bb2ed0)
- (a6d1d9e) [update CHANGELOG](https://github.com/flavorjones/loofah/commit/a6d1d9e307d2c973fe987cdeaca94ded83c850ef)
- (490cd05) [Merge pull request #190 from ahorek/page_break](https://github.com/flavorjones/loofah/commit/490cd05d6ead0d75fceaecd128c4583d9de9ec60)
- (029a496) [version bump to v2.7.0](https://github.com/flavorjones/loofah/commit/029a49636ef64137b40d066633a7836a0ae02a84)


</details>